### ISSUE TITLE
Use "che-e2e:next" instead of "che-e2e:nightly" in Eclipse Che E2E testing CI jobs

### ISSUE THREAD

**Issue by dmytro-ndp** [eclipse-che/che#19905]
### Is your enhancement related to a problem? Please describe.
There is a plan to replace `:nightly` Eclipse Che images with `:next` ones in https://quay.io/organization/eclipse https://github.com/eclipse/che/issues/19291

### Describe the solution you'd like
We need to update Eclipse Che E2E testing CI jobs to use `che-e2e:next` correspondingly. 

### Additional context
["Add nightly and next builds for e2e image" PR](https://github.com/eclipse/che/pull/19907)

**Comment by l0rd** [comment]
isn't that a duplicate of https://github.com/eclipse/che/issues/19905?

**Comment by l0rd** [comment]
oh sorry, I think I got it. This one is about updating the e2e tests after #19905 is completed.

**Comment by dmytro-ndp** [comment]
@l0rd:
> isn't that a duplicate of #19905?

no, it is just the same issue: number #19905
Didn't you confuse it with https://github.com/eclipse/che/issues/19903?
